Source/WebCore/css/CSSStyleRule.cpp

```diff
@@ -45,6 +46,7 @@ CSSStyleRule::CSSStyleRule(StyleRule& styleRule, CSSStyleSheet* parent)
     : CSSRule(parent)
     , m_styleRule(styleRule)
     , m_styleMap(DeclaredStylePropertyMap::create(*this))
+    , m_childRuleCSSOMWrappers(styleRule.nestedRules().size())
```

Comment:
Should we populate this directly instead of doing it in the `item()` function ?  
Maybe this would help to get rid of the `mutable` flag also ?

Reply:
For consistency I was trying to closely match what CSSGroupingRule does to prevent there being any discrepancy in how these lists of rules are created.